File: test/aoc24/day05_test.exs
defmodule Aoc24.Day05Test do
  use ExUnit.Case, async: true

  alias Aoc24.Day05

  describe "part1" do
    test "example" do
      assert Day05.part1("./assets/day05/example.txt") == 143
    end

    test "input" do
      assert Day05.part1("./assets/day05/input.txt") == 7365
    end
  end

  describe "part2" do
    test "example" do
      assert Day05.part2("./assets/day05/example.txt") == 123
    end

    test "input" do
      assert Day05.part2("./assets/day05/input.txt") == 5770
    end
  end
end

File: test/aoc25/day05_test.clj
(ns aoc25.day05-test
  (:require [clojure.test :as t]
            [aoc25.day05 :as day05]))

(t/deftest part1
  (t/testing "Example Input"
    (t/is (= 3 (day05/part1 "example.txt"))))
  (t/testing "Real Input"
    (t/is (= 733 (day05/part1 "input.txt")))))

(t/deftest part2
  (t/testing "Example Input"
    (t/is (= 14 (day05/part2 "example.txt"))))
  (t/testing "Real Input"
    (t/is (= 345821388687084 (day05/part2 "input.txt")))))
